Description: Send move events for wxJoyStick on Unix
 This has been fixed upstream in the 2.9 series, but not in 2.8.12.
Author: Jean GUARDIA <jean.guardia@laposte.net>
Bug-Debian: http://bugs.debian.org/568702
Forwarded: http://trac.wxwidgets.org/ticket/2049
Last-Update: 2011-10-14
Applied-Upstream: 2.9.x

diff --git a/src/unix/joystick.cpp b/src/unix/joystick.cpp
index 411f96a..1deacbc 100644
--- a/wx/src/unix/joystick.cpp
+++ b/wx/src/unix/joystick.cpp
@@ -141,6 +141,15 @@ void* wxJoystickThread::Entry()
                         //       is for some other axes.
                         break;
                 }
+
+                jwx_event.SetPosition(m_lastposition);
+                jwx_event.SetTimestamp(j_evt.time);
+                jwx_event.SetJoystick(m_joystick);
+                jwx_event.SetZPosition(m_axe[3]);
+                jwx_event.SetEventObject(m_catchwin);
+
+                if (m_catchwin)
+                    m_catchwin->AddPendingEvent(jwx_event);
             }
 
             if (j_evt.type & JS_EVENT_BUTTON)
